treetime/treetime.py

```diff
@@ -51,7 +51,29 @@ def __init__(self, *args,**kwargs):
         super(TreeTime, self).__init__(*args, **kwargs)
 
 
-    def run(self, root=None, infer_gtr=True, relaxed_clock=None, n_iqd = None,
+    def run(self, augur=False, **kwargs):
```

Comment:
@victorlin Hi, I modified how the `run` function is called to make sure that the Errors get printed at the correct location when running in `TreeTime` and in `augur` - however @corneliusroemer mentioned that using the argument `augur=true` is not the best coding practice - is there a better way to do this? Perhaps we could use @rneher `raise_uncaught_exception = true` instead as an argument to make it less specific and then add some commentary? Thanks for any input :-)